pkgs/applications/networking/browsers/chromium/update.nix

```diff
@@ -51,6 +51,11 @@ in rec {
       urls = mkUrls "i386";
       sha256 = chanAttrs.sha256bin32;
     } else throw "No Chrome plugins are available for your architecture.");
+
+    toolchain = import ./toolchain.nix {
```

Comment:
Ah, that's for pre-seeding the fixed-output derivation... that's quite fragile TBH and as said, I'd really check what that toolchain fetcher is doing and replicate that behavior instead of hoping that the unsandboxed fetcher will be deterministic.